services/ui-src/src/components/fields/DateField.tsx

```diff
@@ -34,7 +38,8 @@ export const DateField = ({ name, label, sxOverride, ...props }: Props) => {
         name={name}
         label={label}
         onChange={onChangeHandler}
-        value={displayValue || props.hydrate}
+        onBlur={onBlurHandler}
+        value={displayValue || props.hydrate || ""}
```

Comment:
added the empty string here because without it the empty string was not getting set as the initial state and it was throwing a nasty React controlled vs uncontrolled switch error.